server/src/main/java/io/mantisrx/master/jobcluster/job/worker/MantisWorkerMetadataImpl.java

```diff
@@ -186,6 +185,41 @@ public int getNumberOfPorts() {
         return Optional.ofNullable(workerPorts);
     }
 
+    /**
+     * Attempts to build a list of port numbers.
+     *
+     * The difference between this method and simply returning an empty list is that this will at least pass in *some*
+     * value for a port which can be caught by the caller to handle if the caller determines it's invalid.
+     *
+     * Previously, we had code like this:
```

Comment:
this part of the comment is unnecessary IMHO, the git history should have this context.

Reply:
Sounds good, I'll remove.